### PR TITLE
Added UIPageViewController and put rooms list and agenda into that.

### PR DESCRIPTION
**Pull request by kb000**
Note that the transition animation is page flip and I don’t know why.

### CODE REVIEW

**Comment by kb000** [comment]
Fixes [BOOKAROOM-57]

**Comment by kb000** [comment]
screen-flip

File: comeet/UI/MainMenu/MainMenuViewController.swift [hunk 1]
```diff
@@ -226,47 +192,27 @@ private extension MainMenuViewController {
     }
     
     func showAgenda() {
```
Comment (kb000) [inline comeet/UI/MainMenu/MainMenuViewController.swift:194]: This doesn't work when it's triggered post-booking.

File: comeet/UI/MainMenu/MainMenuViewController.swift [hunk 2]
```diff
@@ -105,47 +90,27 @@ extension MainMenuViewController : WWCalendarTimeSelectorProtocol {
...
         
-        addGestureRecognizers()
         dateButton.setTitle(viewModel?.selectedDate.displayStringDate(), for: .normal)
         
-        addRoomsList()
-        addAgendaView()
+        self.agendaVC = getAgendaVC()
+        self.roomListVC = getRoomsListVC()
+        if let agendaVC = self.agendaVC {
+            if let roomListVC = getRoomsListVC() {
```
Comment (ricardocontrerasrobles) [inline comeet/UI/MainMenu/MainMenuViewController.swift:103]: You can link this two unwrappings `if let agendaVC = self.agendaVC, let roomListVC = getRoomsListVC()`

**Comment by ricardocontrerasrobles** [comment]
Nice! Left a very minor comment. I'll follow up to see if we can change page flip transition.

**Pull request merged**